MTGToolbox.Web/Startup.cs

```diff
@@ -19,7 +20,8 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddDbContext<MTGToolboxContext>(options => options.UseSqlServer(Configuration.GetConnectionString("MTGToolboxDB")));
+            services.AddDbContext<MTGToolboxContext>(options => options.UseLazyLoadingProxies()
```

Comment:
This is how you tell EF Core to start using the nuget package to accomplish lazy loading.